exposure-notification/pom.xml

```diff
@@ -4,17 +4,17 @@
 
 	<modelVersion>4.0.0</modelVersion>
 
+    <groupId>fi.thl.covid19</groupId>
```

Comment:
Indentation here too. Is this a github-thing or real difference?

Reply:
Good question. It seems to be fine in my end.

Reply:
No, there is tabs... I'll fix that